schemas/bridging/findIntentsForContextResponse.schema.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://fdc3.finos.org/schemas/next/bridging/findIntentsForContextResponse.schema.json",
+  "title": "FindIntentsForContextResponse",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "bridgeResponse.schema.json"
+    }
+  ],
+  "properties": {
+    "type": {
+      "const": "findIntentsForContextResponse"
+    },
+    "payload": {
+      "type": "object",
+      "properties": {
+        "appIntents": {
+          "type": "array",
+          "items": {
+            "$ref": "../api/appIntent.schema.json"
+          },
+          "additionalProperties": false
+        },
+        "apps": {
+          "type": "array",
+          "items": {
+            "$ref": "../api/appMetadata.schema.json"
+          }
+        }
+      },
+      "additionalProperties": false
+    },
+    "meta": {
+      "allOf": [
+        {
+          "$ref": "bridgeResponse.schema.json#/$defs/ResponseMeta"
+        },
+        {
+          "properties": {
+            "sources": {
+              "type": "array",
+              "items": {
+                "$ref": "../api/desktopAgentIdentifier.schema.json"
+              }
+            }
+          }
+        }
+      ]
```

Comment:
probably need to declare which schema fields come in here... with `destination: false` set

Reply:
Actually may not need  destination: false` that as `ResponseMeta` itself should have that from when it was composed from `RequestMeta`, however, I think you still need the rest of the fields redeclared as `true`